rust/Cargo.lock

```diff
@@ -377,7 +367,7 @@ dependencies = [
  "ff",
  "group",
  "pairing",
- "rand_core 0.6.3",
+ "rand_core",
```

Comment:
Same idea as above, generally I'd try to avoid losing a version number in a change, though I understand this is a `lock` file.